data/json/items/tools.json

```diff
@@ -1139,6 +1139,22 @@
     "max_charges": 500,
     "flags": [ "ALLOWS_REMOTE_USE" ]
   },
+  {
+    "id": "butter_churn",
+    "type": "TOOL",
+    "name": "butter churn",
+    "description": "This is a portable, pedal powered butter churn.  Once filled with raw milk the pedals can be turned to turn it into butter.",
```

Comment:
This churn and "container" churn both have the same names. This might confuse players. I'd recommend somehow distinguish them

Reply:
I changed the this to "folded butter churn". Thoughts?

Reply:
I'd suggest to just remove "butter_churn" item and "f_butter_churn" furniture entry, and make "churn" item into a liquid container item. There's probably little reason to have it as a placeable furniture anyway if it can be used as-is (and it should be usable as-is to reduce tedium, I guess; may assume that time to place down the churn is included into crafting time)

Reply:
> 
> 
> I'd suggest to just remove "butter_churn" item and "f_butter_churn" furniture entry, and make "churn" item into a liquid container item. There's probably little reason to have it as a placeable furniture anyway if it can be used as-is (and it should be usable as-is to reduce tedium, I guess; may assume that time to place down the churn is included into crafting time)

I like furniture items for these sorts of tools that are rather large and would be used in mapgen a lot.  It makes them much easier to place and use, I see it going into a lot of rural mapgen and the faction base camp maps.

Reply:
Well, then at least get rid of "butter_churn" and set furniture to list "churn" as a crafting item that it "provides". As it's done currently, it's just duplicated items.

Reply:
I removed the churn item but realized I had it in tools because the churn furniture uses it as a ``"crafting_pseudo_item": "butter_churn",``  so now the Furniture Churn no longer works.  Should i put the debug only Churn item back in so that the furniture item can call back to it or is there a more elegant solution?